.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,34 @@
+---
+name: ✨ Feature request
+about: Suggest an idea for this project
+---
+
+# Issue Type
+
+- Feature request
+
+# Molecule and Ansible details
```

Comment:
Is this section relevant for FRs at all?

Reply:
Good idea, done